app/models/channel_queue_membership.rb

```diff
@@ -5,5 +5,18 @@ class ChannelQueueMembership < ActiveRecord::Base
   validates_presence_of :user_id, :channel_queue_id
   validates :user_id, uniqueness: { scope: :channel_queue_id }
 
-  scope :active, -> { where('channel_queue_memberships.created_at >= ?', Time.now.in_time_zone('Pacific Time (US & Canada)').beginning_of_day + 7.hours)}
+  scope :active, -> { where('channel_queue_memberships.created_at >= ?', ChannelQueueMembership.queue_open_time)}
+  scope :inactive, -> { where('channel_queue_memberships.created_at < ?', ChannelQueueMembership.queue_open_time)}
+
+  def active?
+    created_at >= ChannelQueueMembership.queue_open_time
+  end
+
+  def inactive?
+    !active?
+  end
+
+  def self.queue_open_time
+    Time.now.in_time_zone('Pacific Time (US & Canada)').beginning_of_day + 7.hours
```

Comment:
I think 8 might be a better start time. Based on the last few weeks the spots never fill up before 8.

Reply:
yeah but they've filled up at like 8:02, and I don't want people to be waiting to queue. What is the downside of 7?

Reply:
Just thinking about the case where someone queues up early in the morning and are first in line when the spots fill.

Reply:
yeah but until we integrate i don't think there's a way to solve that without potentially locking people out

Reply:
this is still an improvement on what we have now